irap-fedora/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nunofonseca/irap_fedora:latest
```

Comment:
Please use the exact tag that we were using, I had to go through many versions to sort of match the versions of tools that we were using on Noah.

Reply:
v1.0.3

Reply:
(or at least to be contemporary to the current noah installation)